docs/user/reference/metrics/text.md

```diff
@@ -0,0 +1,150 @@
+# Text
+
+Records a single long Unicode text, used when the limits on `String` are too low.
+
+{{#include ../../../shared/blockquote-warning.html}}
+
+## Important
+
+> This type should only be used in special cases. See [limitations](#limits) below.
+> It is best to talk to the Glean team before using this.
```

Comment:
Kinda weird for me that we suggest people ask us before using a specific metric type in their own apps.

Reply:
Yeah, I wasn't sure about this. I'm fine with removing that sentence, the limitations should be enough of a "warning"